### ISSUE TITLE
call_extension wrapper

### ISSUE THREAD

**Issue by j91321** [moloch--/sliver-py#21]
**Is your feature request related to a problem? Please describe.**
Not sure if this should be a feature request or bug. The CallExtension and CallExtensionReq have generated grpc code but don't have any wrappers that would expose them by the library.

**Describe the solution you'd like**
Adding sensible call_extension() wrapper. 

**Additional context**
Pretty much required for extensions to work. I'm mostly interested in getting COFF Loader to work.

**Comment by daddycocoaman** [comment]
You're right that it should be exposed. I got ahead of myself generating all the documentation for gRPC for stuff that isn't exposed in sliver-py but should be.
